src/main.rs

```diff
@@ -163,7 +176,91 @@ impl DesignerApp {
                         if let Some(obj) = pool.get_mut_pool().borrow_mut().object_mut_by_id(id) {
                             match obj {
                                 Object::PictureGraphic(o) => {
-                                    // o.load_image(content);
+                                    if let Ok(img) = image::load_from_memory(&content) {
+                                        // Try to preserve alpha channel if available (PNG with transparency)
```

Comment:
I don't see how the transparency logic achieves here? When I upload a picture with white in it, this will just set it to transparent? 

Let's leave it to the user to decide which colour should be transparent with the already available configuration option. We can just map every pixel to it's closest ISOBUS color.